packages/manager/modules/dbaas-logs/src/logs/detail/osd/osd.component.js

```diff
@@ -0,0 +1,8 @@
+import controller from './logs-osd.controller';
+import template from './logs-osd.html';
+
+export default {
+  controller,
+  controllerAs: 'ctrl',
```

Comment:
**suggestion:** Can we consider not using `controllerAs` and replace `ctrl` with `$ctrl` in the template?

Reply:
It looks that this part of the manager (whole LDP section) is full of such approach. I'm not sure it's a good idea to refactor that with this already complex PR.